config.toml

```diff
@@ -192,7 +192,11 @@ ignoreFiles = [
   # These entries appear in the drop-down menu at the top of the website.
   [[params.versions]]
     # <<<<<<<<<<<<<<<<< Update the line below when RELEASING a new minor version 
-    version = "v1.7 (latest)"
+    version = "v1.8 (latest)"
+    githubbranch = "v1.8-release"
+    url = "https://googlecloudplatform.github.io/kubeflow-gke-docs/docs"
+  [[params.versions]]
+    version = "v1.7"
     githubbranch = "v1.7-release"
     url = "https://googlecloudplatform.github.io/kubeflow-gke-docs/docs"
```

Comment:
`url  = "https://googlecloudplatform.github.io/kubeflow-gke-docs/v1.7/docs"`

Reply:
This looks like it should have been updated during 1.7 release. Also shouldn't 1.8 contain `v1.8` in its url as well?